doc/conda-forge.md

```diff
@@ -0,0 +1,113 @@
+# conda-forge based robotology-superbuild Installation
+
+[`conda`](https://docs.conda.io/en/latest/) is a package manager (originally for Python, but effectively language agnostic) that works on Linux, macOS and Windows. 
+
+[`conda-forge`](https://conda-forge.org/)  is a community-mantained channel for the `conda` package manager that provides many dependencies useful for scientific software, in particular all the one that are required by the robotology-superbuild .
+
+## conda-forge overview
+
+The **advantages** of the use of conda and conda-forge are: 
+* **Updated dependencies:** conda-forge tipically has relatively recent version of the dependencies, so if you want to use a recent vtk or pcl (or event a recent compiler) on an old Linux distro such as Ubuntu 16.04, you can. This permits us to support older distributions even when the packages that install via apt are relatively old. 
```

Comment:
Maybe it is worth adding a link to ``vtk`` or ``pcl``, also for the newcomers who may get lost on this point

Reply:
Or maybe also just a word on what they are.

Reply:
Actually, I did not notice this was in a file specific to ``conda``, so I guess this is not really necessary. Feel free to ignore this comment.

Reply:
No, you are right, expanding a bit the sentence and add a link to it is not much effort,  and definitely improves the text.